src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25551.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework.Legacy;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Tests.Issues
+{
+	internal class Issue25551 : _IssuesUITest
+	{
+		public override string Issue => "CollectionView SelectedItem binding issue on initial loading";
+
+		public Issue25551(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		
+		public void CollectionViewPreSelectionShouldUpdate()
+		{
+			App.WaitForElement("ListCollection");
+			VerifyScreenshot();
```

Comment:
There are some pending screenshots (Android etc). You can pick them from the latest build if are correct.

Reply:
@jsuarezruiz , Thank you for the feedback, I have retrieved the images from the recent build and updated the PR accordingly. Please review and let me know if there are any additional concerns.